go/porcelain/deploy.go

```diff
@@ -580,7 +585,7 @@ func walk(dir string, observer DeployObserver, useLargeMedia, ignoreInstallDirs
 	return files, err
 }
 
-func bundle(functionDir string, observer DeployObserver) (*deployFiles, []models.FunctionSchedule, error) {
+func bundle(ctx context.Context, functionDir string, observer DeployObserver) (*deployFiles, []*models.FunctionSchedule, error) {
```

Comment:
`bundle` and `bundleFromManifest` are now receiving the context so that they can acquire a logger and print a debug message.